pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/PicklistQuestionDao.java

```diff
@@ -146,10 +147,30 @@ default long insertOption(long questionId, PicklistOptionDef option, int display
             detailLabelTmplId = option.getDetailLabelTemplate().getTemplateId();
         }
 
+        Long nestedOptionsTmplId = null;
+        if (option.getNestedOptionsLabelTemplate() != null) {
+            templateDao.insertTemplate(option.getNestedOptionsLabelTemplate(), revisionId);
+            nestedOptionsTmplId = option.getNestedOptionsLabelTemplate().getTemplateId();
+        }
+
         long optionId = jdbiOption.insert(questionId, option.getStableId(), optionLabelTmplId, tooltipTmplId,
-                detailLabelTmplId, option.isDetailsAllowed(), option.isExclusive(), displayOrder, revisionId);
+                detailLabelTmplId, option.isDetailsAllowed(), option.isExclusive(), displayOrder, revisionId, nestedOptionsTmplId);
         option.setOptionId(optionId);
 
+        if (CollectionUtils.isNotEmpty(option.getNestedPicklistOptions())) {
+            int subOptionDisplayOrder = displayOrder;
```

Comment:
🤔 If we use the same `displayOrder` as the parent option and incrementing it for each nested option, wouldn't we overlap with the display order for other top-level options? Should we start fresh with the display order here?

Reply:
I am starting nested option display orders with parent option order  + 10; It helps with some existing display-order order by queries. 
Overlap with other parent options is ok because order matters within parents or nested only.

Reply:
I guess that's fine for now.